knimin/lib/mem_zip.py

```diff
@@ -61,6 +61,42 @@ def write_to_buffer(self):
         return self.in_memory_data.getvalue()
 
 
+def extract_zip(input_zip):
```

Comment:
for sure, if you extract a huge archive. But it need to do the extraction for unit testing and I thought it is more elegant than doing system calls to 'unzip'